common/encoding.h

```diff
@@ -96,134 +96,137 @@ class Encoding {
 		}
 
 		/**
-		 * @return The encoding, which is currently being converted from
+		 * @return The encoding that is currently being converted from.
 		 */
 		String getFrom() {return _from;};
 
 		/**
-		 * @param from The encoding, to convert from
+		 * @param from The encoding to convert from.
 		 */
 		void setFrom(const String &from) {_from = from;};
 
 		/**
-		 * @return The encoding, which is currently being converted to
+		 * @return The encoding that is currently being converted to.
 		 */
 		String getTo() {return _to;};
 
 		/**
-		 * @param to The encoding, to convert to
+		 * @param to The encoding to convert to.
 		 */
 		void setTo(const String &to) {_to = to;};
 
 		/**
-		 * Switches the endianity of a string.
+		 * Switch the endianness of a string.
 		 *
-		 * @param string Array containing the characters of a string.
-		 * @param length Length of the string in bytes
+		 * @param string   Array containing the characters of a string.
+		 * @param length   Length of the string in bytes.
 		 * @param bitCount Number of bits used for each character.
 		 *
-		 * @return Array of characters with the opposite endianity
+		 * @return Array of characters with the opposite endianness.
 		 */
 		static char *switchEndian(const char *string, int length, int bitCount);
 
 		/**
-		 * Computes length (in bytes) of a string in a given encoding. 
+		 * Compute the length (in bytes) of a string in a given encoding.
+		 *
 		 * The string must be zero ended. Similar to strlen
-		 * (could be used instead of strlen).
+		 * (can be used instead of strlen).
```

Comment:
Maybe it should be clarified that this **must** be used instead of `strlen` for some encodings (such as UTF-16 and UTF-32) as `strlen` does not support multi-bytes encodings, with some exceptions (such as UTF-8).